sbnobj/Common/Reco/CRUMBSResult.h

```diff
@@ -0,0 +1,105 @@
+// Class for holding the result of the CRUMBS slice ID
+
+#ifndef sbncode_CRUMBSResult_H
+#define sbncode_CRUMBSResult_H
+
+namespace sbn {
+  class CRUMBSResult {
+  public:
+
+    CRUMBSResult() 
+      : score(-5.f)
+      , tpc_CRFracHitsInLongestTrack(-5.f)
+      , tpc_CRLongestTrackDeflection(-5.f)
+      , tpc_CRLongestTrackDirY(-5.f)
+      , tpc_CRNHitsMax(-5.f)
+      , tpc_NuEigenRatioInSphere(-5.f)
+      , tpc_NuNFinalStatePfos(-5.f)
+      , tpc_NuNHitsTotal(-5.f)
+      , tpc_NuNSpacePointsInSphere(-5.f)
+      , tpc_NuVertexY(-5.f)
+      , tpc_NuWeightedDirZ(-5.f)
+      , tpc_StoppingChi2CosmicRatio(-5.f)
+      , pds_FMTotalScore(-5.f)
+      , pds_FMPE(-5.f)
+      , pds_FMTime(-5.f)
+      , crt_TrackScore(-5.f)
+      , crt_HitScore(-5.f)
+      , crt_TrackTime(-5.f)
+      , crt_HitTime(-5.f)
+    {
+    }
+    
+    CRUMBSResult(const float score)
+      : score(score)
+      , tpc_CRFracHitsInLongestTrack(-5.f)
+      , tpc_CRLongestTrackDeflection(-5.f)
+      , tpc_CRLongestTrackDirY(-5.f)
+      , tpc_CRNHitsMax(-5.f)
+      , tpc_NuEigenRatioInSphere(-5.f)
+      , tpc_NuNFinalStatePfos(-5.f)
+      , tpc_NuNHitsTotal(-5.f)
+      , tpc_NuNSpacePointsInSphere(-5.f)
+      , tpc_NuVertexY(-5.f)
+      , tpc_NuWeightedDirZ(-5.f)
+      , tpc_StoppingChi2CosmicRatio(-5.f)
+      , pds_FMTotalScore(-5.f)
+      , pds_FMPE(-5.f)
+      , pds_FMTime(-5.f)
+      , crt_TrackScore(-5.f)
+      , crt_HitScore(-5.f)
+      , crt_TrackTime(-5.f)
+      , crt_HitTime(-5.f)
```

Comment:
Can this be
```suggestion
      : CRUMBResult(),
        score(score)
```

Reply:
Kind of, have to change the order and then it calls it implicitly. I definitely learned a few things looking at this :D

Reply:
Are you sure? I'm pretty sure the score-only constructor you have right now will leave the other fields initialized with junk. Did my suggestion not compile?

Actually, I'm now remembering what Gianluca would say about all this, which would be to recommend you give these default values inline in the header file. Then they do implicitly apply to all constructors.